### PR TITLE
Add Keycloak to Easypanel 🎉 

### PR DESCRIPTION
**Pull request by Supernova3339**

### CODE REVIEW

File: templates/keycloak/index.ts [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+      serviceName: input.appServiceName,
+      env: [
+        `KEYCLOAK_DATABASE_HOST=${input.projectName}_${input.databaseServiceName}`,
+        `KEYCLOAK_DATABASE_NAME=${input.databaseServiceName}`,
+        `KEYCLOAK_DATABASE_USER=postgres`,
+        `KEYCLOAK_DATABASE_PASSWORD=${databasePassword}`,
+        `KEYCLOAK_ADMIN_USER=${input.keycloakUsername}`,
+        `KEYCLOAK_ADMIN_PASSWORD=${input.keycloakPassword}`,
+        `KC_HOSTNAME_STRICT_HTTPS=false`,
+        `KC_HOSTNAME=${input.domain}`,,
```
Comment (Ponkhy) [inline templates/keycloak/index.ts:25]:
```suggestion
        `KC_HOSTNAME=${input.domain}`,
```

File: templates/keycloak/index.ts [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+  const databasePassword = randomPassword();
+
+  services.push({
+    type: "app",
+    data: {
+      projectName: input.projectName,
+      serviceName: input.appServiceName,
+      env: [
+        `KEYCLOAK_DATABASE_HOST=${input.projectName}_${input.databaseServiceName}`,
+        `KEYCLOAK_DATABASE_NAME=${input.databaseServiceName}`,
```
Comment (Ponkhy) [inline templates/keycloak/index.ts:19]:
```suggestion
        `KEYCLOAK_DATABASE_NAME=${input.projectName}`,
```

Comment (Supernova3339) [reply templates/keycloak/index.ts:19]: oh whoops

**Review by Ponkhy — APPROVED** [review]
LGTM

**Pull request merged**